tools/build-local-flatpak.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Convenience script to build a Flatpak bundle with the local contents (the original manifest)
+# builds from git so changes need to be committed.
+#
+# Copyright (c) 2018 Endless Mobile Inc.
+#
+# Author: Joaquim Rocha <jrocha@endlessm.com>
+#
+
+set -e
+source_dir="$(git rev-parse --show-toplevel)"
+
+pushd "$source_dir"
+
+# If the GIT_CLONE_BRANCH var is not defined, we replace it so that it'll in fact set the
+# git branch to "" but add also "type": "dir" so it builds from the current directory.
+# This is hacky but until we need to keep git as the source type, then it's the less intrusive.
+GIT_CLONE_BRANCH=${GIT_CLONE_BRANCH:-'", "type": "dir'}
```

Comment:
I was not really a fan of this in the clubhouse repo either - it prints out a big warning whenever you build it. But it is harmless, I guess we should do it for now and keep working on a better solution in the meantime.